web/src/main/java/org/fao/geonet/Geonetwork.java

```diff
@@ -145,6 +149,10 @@ public Object start(Element config, ServiceContext context) throws Exception {
         context.setAsThreadLocal();
         this._applicationContext = context.getApplicationContext();
         ApplicationContextHolder.set(this._applicationContext);
+        
+        // refresh log from db
+        LogUtils logUtils = new LogUtils(context);
```

Comment:
When the app starts, I get a NPE:
 
```
2015-04-20 12:27:35,764 ERROR [jeeves.engine] - {Stack=java.lang.NullPointerException
	at org.fao.geonet.services.config.LogUtils.<init>(LogUtils.java:23)
	at org.fao.geonet.Geonetwork.start(Geonetwork.java:154)
	at jeeves.server.JeevesEngine.initAppHandler(JeevesEngine.java:452)
```

Probably because gnContext is not yet set.